src/main/java/model/producer/TruckProducer.java

```diff
@@ -0,0 +1,21 @@
+package model.producer;
+
+import core.mate.academy.model.Truck;
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer {
+
+    @Override
+    public List<Truck> get() {
```

Comment:
The return type of the `get()` method should be parameterized to match the `MachineProducer` interface, which should be `List<Truck>` in this case to prevent raw types usage.